mifosng-android/src/main/res/values/strings.xml

```diff
@@ -245,6 +246,18 @@
     <string name="menu_run_report">Run Report</string>
     <string name="menu_export_csv">Export CSV</string>
     <string name="msg_report_empty">No data retrieved for the chosen report!</string>
+    <string name="report_loan_purpose">"Loan Purpose"</string>
+    <string name="par_calculation">PAR Calculation</string>
+    <string name="savings_acc_deposit">SavingsAccDeposit</string>
+    <string name="glaccount">GLAccountNO</string>
+    <string name="obligation_date_type">Obligation date type</string>
+    <string name="start_date">Start Date</string>
```

Comment:
Please verify if these are already not present. There are chances that these may already be used somewhere. Ignore if they are not present earlier.

Reply:
@tarun0  verified